res/values-ga/strings.xml

```diff
@@ -26,7 +26,7 @@
 
     <string name="Are_You_Sure_">"An bhfuil tú cinnte?"</string>
     <string name="Yes">"Tá"</string>
-    <string name="No">"No:"</string>
+    <string name="No">"Níl:"</string>
```

Comment:
I know the colon : has been kept, but please remove it, it needs to be based on english values.